packages/datadog-instrumentations/src/aws-sdk.js

```diff
@@ -155,6 +155,8 @@ function getMessage (request, error, result) {
 }
 
 function getChannelSuffix (name) {
+  // some resource identifiers have spaces between ex: bedrock runtime
+  name = name.split(' ').join('')
```

Comment:
```suggestion
  name = name.replaceAll(' ', '')
```
This should be quicker as it skips the intermediary array representation.